yb-voyager/src/srcdb/pg_dump_export_data.go

```diff
@@ -45,10 +45,9 @@ func pgdumpExportDataOffline(ctx context.Context, source *Source, connectionUri
 
 	pgDumpArgs := fmt.Sprintf(`--no-blobs --data-only --no-owner --compress=0 %s -Fd --file %s --jobs %d --no-privileges --no-tablespaces --load-via-partition-root`,
 		tableListPatterns, dataDirPath, source.NumConnections)
+	os.Setenv("PGPASSWORD", source.Password)
 	cmd := fmt.Sprintf(`%s '%s' %s`, pgDumpPath, connectionUri, pgDumpArgs)
```

Comment:
Oh! The password is inside the `connectionUri` :-) You will have to generate a new URI without a password in it and then pass it to the command. You can implement a function like `removePasswordFromUri(uri string) string)`.

Reply:
Ah! My bad! I didn't check the last file.